gui/default/index.html

```diff
@@ -647,6 +652,7 @@ <h4 class="panel-title">
   <ng-include src="'syncthing/core/shutdownDialogView.html'"></ng-include>
   <ng-include src="'syncthing/device/idqrModalView.html'"></ng-include>
   <ng-include src="'syncthing/device/editDeviceModalView.html'"></ng-include>
+  <ng-include src="'syncthing/device/changeLogModalView.html'"></ng-include>
```

Comment:
This is a bit inconsistent as well, globalDiskChanges vs LastLocalChagne vs changeLog